src/app/devices/deviceContent/components/directMethod/__snapshots__/directMethod.spec.tsx.snap

```diff
@@ -61,6 +61,7 @@ exports[`directMethod matches snapshot 1`] = `
                 "automaticLayout": true,
               }
             }
+            theme="vs-dark"
```

Comment:
I find it mysterious that the last commit which made light theme as default, these snap shot are still passing

Reply:
Especially since the required theme prop wasn't being passed. Thanks for catching things jest wasn't throwing on.